.storybook/preview.ts

```diff
@@ -5,6 +5,19 @@ import '../src/app/index.css';
 const preview: Preview = {
   parameters: {
     actions: { argTypesRegex: '^on[A-Z].*' },
+    backgrounds: {
+      default: 'leather-light-mode',
+      values: [
```

Comment:
Maybe we can take these values from the `tokens` ?

Reply:
Yeah, I meant to come back here to check if works, will try.

Reply:
I used them for the [storybook custom viewports](https://github.com/leather-wallet/extension/blob/92f3ffadea27d66555fa757384c47927f840fc59/.storybook/viewports.ts) so it should be fine

Reply:
I can't get the import to work here. 🤔

Reply:
OK let's forget it for now. I can check if I can get it working. Probably some weird `monorepo` issue we need to improve anyway